dlt/assets.py

```diff
@@ -21,7 +22,7 @@ def sourcename_resource(api_secret_key=dlt.secrets.value):
     print(headers)
 
     # make an api call here
-    url = "https://www49.myfantasyleague.com/2024/export?TYPE=assets&L=59643&APIKEY=ahFi18iVvuWrx1GmPVDHZTEeF7ox&JSON=1"
+    url = f"https://www49.myfantasyleague.com/{config.league_year}/export?TYPE=assets&L={config.league_id}&APIKEY={config.mfl_api_key}&JSON=1"
```

Comment:
one other thing to consider. their host prefix is not necessarily constant from year to year - i know this from experience. for instance, i don't know where `www49` comes in... and some years that'll change to something else without explanation.

so perhaps it is good to create another var for `config.host` whose value is `www49.myfantasyleague.com` (at least currently) - that way a future change to this would be only a config change and not a config change and code deploy.